app/services/katello/pulp3/api/content_guard.rb

```diff
@@ -0,0 +1,35 @@
+require 'pulp_certguard_client'
+
+module Katello
+  module Pulp3
+    module Api
+      class ContentGuard < Core
+        def self.client_module
+          PulpCertguardClient
+        end
+
+        def api_client
+          PulpCertguardClient::ApiClient.new(smart_proxy.pulp3_cert_configuration(PulpCertguardClient::Configuration))
+        end
+
+        def rhsm_api_client
+          PulpCertguardClient::ContentguardsRHSMApi.new(api_client)
+        end
+
+        def create(name = "RHSMCertGuard")
+          data = PulpCertguardClient::CertguardRHSMCertGuard.new(name: name, ca_certificate: Cert::Certs.ca_cert)
+          rhsm_api_client.create(data)
```

Comment:
we probably want to handle a create conflict here, in case two get created with the same name in pulp, the conflict will be 'ignored' and the initial will be recorded/used

Reply:
i think this is still unresolved?

Reply:
Verfied that name is indeed unique for contentguard. So catching exception should work.